src/main.py

```diff
@@ -11,6 +11,9 @@
 
 pg.init()  # pylint: disable=no-member
 
+#Variable
+Jump = False
```

Comment:
- Follow the snake_case naming convention and lower the first letter
- Type annotations are good python practice to use to keep consistency
```suggestion
            jump: bool = False
            sneak: bool = False
```